### PR TITLE
[JBIDE-23640] DConnection#getImage by tag should inspect all tags

### PR DESCRIPTION
**Pull request by adietish**
* fixed DockerConnection¼getImage(name, tag) to inspect all tags
* fixed test to wait for push to be finished

### CODE REVIEW

**Comment by adietish** [comment]
@jkopriva please review. It contains the changes that we discussed this afternoon

File: plugins/org.jboss.tools.docker.reddeer/src/org/jboss/tools/docker/reddeer/condition/ImageIsDeployedCondition.java [hunk 1]
```diff
@@ -0,0 +1,36 @@
+package org.jboss.tools.docker.reddeer.condition;
```
Comment (jkopriva) [inline plugins/org.jboss.tools.docker.reddeer/src/org/jboss/tools/docker/reddeer/condition/ImageIsDeployedCondition.java]: @adietish Please add copyright

**Comment by adietish** [comment]
@jkopriva ok, done.

**Comment by jkopriva** [comment]
testPR

**Pull request merged**